.github/workflows/ios.yml

```diff
@@ -14,9 +14,16 @@ jobs:
   build_ios:
   name: build-ios
     runs-on: macos-latest
-    unity_version: ${{ github.event.inputs.unity_version }}
-    unity_install_dir: /Applications/Unity_${{ github.event.inputs.unity_version }}
-    build_dir: ios_unity
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macos-latest]
```

Comment:
There is only one value here. Do you plan to expanded the value in the future?

Reply:
hmmm no, but not sure what's the best way to put extra attributes like unity version etc. The current setup has issue.